build.sbt

```diff
@@ -1,19 +1,18 @@
-
 import play.sbt.routes.RoutesKeys
 import sbt.*
-import sbt.Keys.libraryDependencySchemes
-import uk.gov.hmrc.DefaultBuildSettings.*
+import sbt.Keys.*
+import scoverage.ScoverageKeys
+import uk.gov.hmrc.DefaultBuildSettings.{defaultSettings, scalaSettings}
 import uk.gov.hmrc.versioning.SbtGitVersioning.autoImport.majorVersion
-import uk.gov.hmrc.DefaultBuildSettings
 
 val appName = "income-tax-view-change-frontend"
 
 val bootstrapPlayVersion = "8.1.0"
 val playPartialsVersion = "9.1.0"
 val playFrontendHMRCVersion = "8.5.0"
-val catsVersion = "2.8.0"
+val catsVersion = "2.12.0"
```

Comment:
If we can kindly bundle any library upgrade into a separate story to have some level of control where we are heading etc;